package.json

```diff
@@ -8,7 +8,7 @@
     "test:watch": "nodemon -q -x 'npm test'"
   },
   "engines": {
-    "node": ">= 6.0.0"
+    "node": ">= 8.0.0"
```

Comment:
Technically, this is a breaking change, however the previous CI had already bumped to test in node@10+. 8 is still passing, but maybe I should bump it to something even more modern?

I figure node@6 might have been only failing because it is using npm@3 by default, which does not yet have `package-lock.json`, so I could try to address that in `pkgjs/action` (by installing newer npm), and then we could try reverting this change for supported node?

LMK what's the preferred course.

Reply:
No worries. Guess we've just forgotten to bump this when we've upgraded on the github-bot server.

ATM we're running Node.js 14, so although this version range isn't necessarily wrong, it's surely misleading.

> .. but maybe I should bump it to something even more modern?

Yeah, sounds like a good idea to me. Having the above in mind, having `>= 14.0.0` would make sense, right?